src/transformers/audio_utils.py

```diff
@@ -363,6 +380,8 @@ def spectrogram(
         db_range (`float`, *optional*):
             Sets the maximum dynamic range in decibels. For example, if `db_range = 80`, the difference between the
             peak value and the smallest value will never be more than 80 dB. Must be greater than zero.
+        remove_dc_offset (`bool`, *optional*):
+            Subtract mean from waveform on each frame, applied before pre-emphasis.
```

Comment:
let's also mention that this is used for torch audio and we recommend setting it to True no?